packages/language-server/src/plugins/typescript/features/CodeActionsProvider.ts

```diff
@@ -110,29 +109,17 @@ export class CodeActionsProviderImpl implements CodeActionsProvider {
             return [];
         }
 
-        const useSemicolons =
-            this.configManager.getMergedPrettierConfig(
-                await importPrettier(document.getFilePath()!).resolveConfig(
-                    document.getFilePath()!,
-                    {
-                        editorconfig: true
-                    }
-                )
-            ).semi ?? true;
-        const documentUseLf =
-            document.getText().includes('\n') && !document.getText().includes('\r\n');
-
         const changes = lang.organizeImports(
             {
                 fileName: tsDoc.filePath,
                 type: 'file',
                 skipDestructiveCodeActions
             },
             {
-                newLineCharacter: documentUseLf ? '\n' : ts.sys.newLine,
-                semicolons: useSemicolons
-                    ? ts.SemicolonPreference.Insert
-                    : ts.SemicolonPreference.Remove
+                ...(await this.configManager.getFormatCodeSettingsForFile(document)),
+
+                // handle it on our own
+                baseIndentSize: undefined
```

Comment:
`baseIndent` has some trouble here. Since we don't have indents for import in the svelte2tsx. So it would make it always have organize-import. Think it would make more sense to it on our own.